container/root/etc/cont-init.d/10-nginx-config.sh

```diff
@@ -33,6 +33,50 @@ then
   sed -i "s/\#gzip/gzip/" $CONF_NGINX_SERVER
 fi
 
+if [[ $SERVER_ENABLE_BROTLI ]]
+then
+  echo "[nginx] enabling brotli"
+  TYPES_TO_COMPRESS="application\/javascript application\/json application\/rss+xml application\/vnd.ms-fontobject application\/x-font application\/x-font-opentype application\/x-font-otf application\/x-font-truetype application\/x-font-ttf application\/xhtml+xml application\/xml font\/opentype font\/otf font\/ttf image\/svg+xml image\/x-icon text\/css text\/javascript text\/plain text\/xml;"
+  ngver=$(nginx -v 2>&1 | grep -o '[0-9\.]*')
+
+  # To automatically select NGINX modules directory:
+  [ -d /usr/share/nginx/modules ] && moddir=/usr/share/nginx/modules
+  [ -d $(nginx -V 2>&1 | grep -o 'prefix=[^ ]*' | sed 's/prefix=//')/modules ] && moddir="$(nginx -V 2>&1 | grep -o 'prefix=[^ ]*' | sed 's/prefix=//')/modules"
+  [ -d $(nginx -V 2>&1 | grep -o 'modules-path=[^ ]*' | sed 's/modules-path=//') ] && moddir="$(nginx -V 2>&1 | grep -o 'modules-path=[^ ]*' | sed 's/modules-path=//')"
+
+  # Set temporary directory and build on it
+  builddir="$(mktemp -d)"
+  cd "${builddir}"
+
+  # Download and unpack NGINX
+  wget https://nginx.org/download/nginx-${ngver}.tar.gz && { tar zxf nginx-${ngver}.tar.gz && rm nginx-${ngver}.tar.gz; } || { echo '!! download failed, exiting...'; exit 2; }
+
+  # Download, initialize, and make Brotli dynamic modules
+  git clone https://github.com/google/ngx_brotli.git
```

Comment:
instead of cloning and getting the latest from master, what do you think about using a release? https://github.com/google/ngx_brotli/releases